### PR TITLE
Update requirements for Data Jobs Monitoring EMR setup page

### PR DESCRIPTION
**Pull request by yiliangzhou**
### What does this PR do? What is the motivation?

Data Jobs Monitoring updated its setup process to support earlier EMR releases (from EMR release `6.0.1` and later), update the documentation to reflect this in the requirement section of the setup page.

The setup guide for EMR of Data Jobs Monitoring is missing instruction for required permissions, and the order for configuring the permission is wrong and needs to be moved before the creation of the cluster.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by gabe-ochoa — APPROVED** [review]

File: content/en/data_jobs/emr.md [hunk 1]
```diff
@@ -33,6 +33,52 @@ Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
...
+1. On the next page, under the **Permissions** tab, find the **Permissions policies** section. Click on **Add permissions** > **Create inline policy**.
+1. On the **Specify permissions** page, find the **Select a service** section. Under **Service**, select **Secrets Manager**.
+   {{< img src="data_jobs/emr/specify_permissions.png" alt="AWS IAM console, Specify Permissions page." style="width:80%;" >}}
+   - Then, under **Actions allowed**, select `GetSecretValue`. This is a **Read** action.
+   - Under **Resources**, select **Specific**. Then, next to **Secret**, click on **Add ARNs** and add the ARN of the secret you created in the first step on this page.
+   - Click **Next**.
+1. On the next page, give your policy a name. Then, click **Create policy**.
+
+#### Permissions to describe cluster
+You can skip this step if you are using the default role **EMR_EC2_DefaultRole** for cluster EC2 instances and its associated AWS default managed policy, **AmazonElasticMapReduceforEC2Role**.
```
Comment (warrierr) [inline content/en/data_jobs/emr.md:51]:
```suggestion
Only do this step if you are NOT using the default role **EMR_EC2_DefaultRole** for cluster EC2 instances and its associated AWS default managed policy, **AmazonElasticMapReduceforEC2Role**.
```

**Review by warrierr — CHANGES_REQUESTED** [review]

**Comment by drichards-87** [comment]
Created DOCS-8472 for Docs Team review.

**Review by warrierr — APPROVED** [review]
LGTM

File: content/en/data_jobs/emr.md [hunk 2]
```diff
@@ -33,6 +33,52 @@ Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
 1. On the **Review** page, review your secret details. Then, click **Store**.
 1. In AWS Secrets Manager, open the secret you created. Take note of the **Secret ARN**.
 
+### Grant permissions to EMR EC2 instance profile
+EMR EC2 instance profile is a IAM role assigned to every EC2 instance in an Amazon EMR cluster when the instance launches, follow [the offical guide][11] to prepare this role based on your application's need to interact with other AWS services. The following permssions are additional permissions required for Data Jobs Monitoring.
+
+#### Permissions to get secret value using AWS Secret Management (Recommended)
```
Comment (warrierr) [inline content/en/data_jobs/emr.md:39]:
```suggestion
#### Permissions to get secret value using AWS Secret Management (required if using Secrets Manager)
```

File: content/en/data_jobs/emr.md [hunk 3]
```diff
@@ -33,6 +33,52 @@ Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
 1. On the **Review** page, review your secret details. Then, click **Store**.
 1. In AWS Secrets Manager, open the secret you created. Take note of the **Secret ARN**.
 
+### Grant permissions to EMR EC2 instance profile
+EMR EC2 instance profile is a IAM role assigned to every EC2 instance in an Amazon EMR cluster when the instance launches, follow [the offical guide][11] to prepare this role based on your application's need to interact with other AWS services. The following permssions are additional permissions required for Data Jobs Monitoring.
```
Comment (warrierr) [inline content/en/data_jobs/emr.md:37]:
```suggestion
EMR EC2 instance profile is a IAM role assigned to every EC2 instance in an Amazon EMR cluster when the instance launches, follow [the offical guide][11] to prepare this role based on your application's need to interact with other AWS services. The following additional permssions may be required for Data Jobs Monitoring.
```

File: content/en/data_jobs/emr.md [hunk 4]
```diff
@@ -33,6 +33,52 @@ Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
...
+1. Click on the IAM role you plan to use as the instance profile for your EMR cluster.
+1. On the next page, under the **Permissions** tab, find the **Permissions policies** section. Click on **Add permissions** > **Create inline policy**.
+1. On the **Specify permissions** page, find the **Select a service** section. Under **Service**, select **Secrets Manager**.
+   {{< img src="data_jobs/emr/specify_permissions.png" alt="AWS IAM console, Specify Permissions page." style="width:80%;" >}}
+   - Then, under **Actions allowed**, select `GetSecretValue`. This is a **Read** action.
+   - Under **Resources**, select **Specific**. Then, next to **Secret**, click on **Add ARNs** and add the ARN of the secret you created in the first step on this page.
+   - Click **Next**.
+1. On the next page, give your policy a name. Then, click **Create policy**.
+
+#### Permissions to describe cluster
```
Comment (warrierr) [inline content/en/data_jobs/emr.md:50]:
```suggestion
#### Permissions to describe cluster (required if NOT using the default role EMR_EC2_DefaultRole)
```

File: content/en/data_jobs/emr.md [hunk 5]
```diff
@@ -33,6 +33,52 @@ Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
...
+Only do this step if you are NOT using the default role **EMR_EC2_DefaultRole** for cluster EC2 instances and its associated AWS default managed policy, **AmazonElasticMapReduceforEC2Role**.
```
Comment (warrierr) [inline content/en/data_jobs/emr.md:51]:
```suggestion
```

**Review by warrierr — CHANGES_REQUESTED** [review]

File: content/en/data_jobs/emr.md [hunk 6]
```diff
@@ -33,6 +33,51 @@ Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
 1. On the **Review** page, review your secret details. Then, click **Store**.
 1. In AWS Secrets Manager, open the secret you created. Take note of the **Secret ARN**.
 
+### Grant permissions to EMR EC2 instance profile
+EMR EC2 instance profile is a IAM role assigned to every EC2 instance in an Amazon EMR cluster when the instance launches, follow [the offical guide][11] to prepare this role based on your application's need to interact with other AWS services. The following additional permssions may be required for Data Jobs Monitoring.
```
Comment (cswatt) [inline content/en/data_jobs/emr.md:37]:
```suggestion
EMR EC2 instance profile is a IAM role assigned to every EC2 instance in an Amazon EMR cluster when the instance launches. Follow [the Amazon guide][11] to prepare this role based on your application's need to interact with other AWS services. The following additional permissions may be required for Data Jobs Monitoring.
```

File: content/en/data_jobs/emr.md [hunk 7]
```diff
@@ -33,6 +33,51 @@ Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
...
+
+#### Permissions to get secret value using AWS Secret Management (required if using Secrets Manager)
```
Comment (cswatt) [inline content/en/data_jobs/emr.md:39]:
```suggestion
#### Permissions to get secret value using AWS Secrets Manager

<div class="alert alert-warning">
These permissions are <strong>required</strong> if you are using AWS Secrets Manager.
</div>
```

File: content/en/data_jobs/emr.md [hunk 8]
```diff
@@ -33,6 +33,51 @@ Follow these steps to enable Data Jobs Monitoring for Amazon EMR.
...
+1. Click on the IAM role you plan to use as the instance profile for your EMR cluster.
+1. On the next page, under the **Permissions** tab, find the **Permissions policies** section. Click on **Add permissions** > **Create inline policy**.
+1. On the **Specify permissions** page, find the **Select a service** section. Under **Service**, select **Secrets Manager**.
+   {{< img src="data_jobs/emr/specify_permissions.png" alt="AWS IAM console, Specify Permissions page." style="width:80%;" >}}
+   - Then, under **Actions allowed**, select `GetSecretValue`. This is a **Read** action.
+   - Under **Resources**, select **Specific**. Then, next to **Secret**, click on **Add ARNs** and add the ARN of the secret you created in the first step on this page.
+   - Click **Next**.
+1. On the next page, give your policy a name. Then, click **Create policy**.
+
+#### Permissions to describe cluster (required if NOT using the default role EMR_EC2_DefaultRole)
```
Comment (cswatt) [inline content/en/data_jobs/emr.md:50]:
```suggestion
#### Permissions to describe cluster

<div class="alert alert-warning">
These permissions are <strong>required</strong> if you are <strong>NOT</strong> using the default role, <code>EMR_EC2_DefaultRole</code>.
</div>
```

File: content/en/data_jobs/emr.md [hunk 9]
```diff
@@ -60,35 +105,17 @@ When you create a new EMR cluster in the [Amazon EMR console][4], add a bootstra
 
    ```
 
-   The script above sets the required parameters, downloads and runs the latest init script for Data Jobs Monitoring in EMR. If you want to pin your script to a specific version, you can replace the file name in the URL with `emr_init_1.3.1.sh` to use the last stable version.
+   The script above sets the required parameters, downloads and runs the latest init script for Data Jobs Monitoring in EMR. If you want to pin your script to a specific version, you can replace the file name in the URL with `emr_init_1.4.0.sh` to use the last stable version.
```
Comment (cswatt) [inline content/en/data_jobs/emr.md:108]:
```suggestion
   The script above sets the required parameters, and downloads and runs the latest init script for Data Jobs Monitoring in EMR. If you want to pin your script to a specific version, you can replace the file name in the URL with `emr_init_1.4.0.sh` to use the last stable version.
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**